src/biogeochem/CNFireLi2024Mod.F90

```diff
@@ -593,7 +589,6 @@ subroutine CNFireArea (this, bounds, num_soilc, filter_soilc, num_soilp, filter_
            end if
            fuelc(c) = fuelc(c)/(1._r8-cropf_col(c))
            fb       = max(0.0_r8,min(1.0_r8,(fuelc(c)-lfuel)/(ufuel-lfuel)))
-           if (trotr1_col(c)+trotr2_col(c)<=0.6_r8) then
```

Comment:
Does the removal of this means indentation should be realigned?

Reply:
Done with bedff9cb7.